cli/inspector.rs

```diff
@@ -394,7 +394,7 @@ impl DenoInspector {
     server: Option<Arc<InspectorServer>>,
   ) -> Box<Self> {
     let context = isolate.global_context();
-    let scope = &mut v8::HandleScope::new(&mut **isolate);
+    let scope = &mut v8::HandleScope::new(isolate.v8_isolate());
```

Comment:
Not now, but we should eliminate variables called "isolate" in CLI where they in fact refer to JsRuntimes

Reply:
I agree, it's on my list